documentation/source/user_guide_model_creation.rst

```diff
@@ -529,4 +529,240 @@ The following image shows the simulation results:
+
+.. code-block:: json
+
+    {
+      "single_sweeps": [
+        {
+          "f": 200000,
+          "winding1": {
+            "turn_losses": [...],
+            "flux": [6.34870443074174e-06, -6.969982393761393e-07],
+            "flux_over_current": [3.17434773053773e-06, -3.51948446513906e-07],
+            "V": [0.8845429232083418, 7.978006008157411],
+            ...
+          },
+          "winding2": {
+            ...
+          },
+          "core_eddy_losses": 0.00050908155779138,
+          "core_hyst_losses": 3.16018326710339,
+          ...
+          "all_winding_losses": 0.5355581006243983
+        }
+      ],
+      "total_losses": {
+        "winding1": {
+          "total": 0.5355581006244025,
+           "turns": [...]
+        }
+        "all_windings": 0.5355581006243983,
+        "eddy_core": 0.09956183619015413,
+        "hyst_core_fundamental_freq": 3.16018326710339,
+        "total_core_part_1": 3.259745103293544,
+        "total_eddy_core_part_1": 0.09956183619015413,
+        "total_hyst_core_part_1": 3.16018326710339,
+        "core": 3.259745103293544,
+        "total_losses": 3.7953032039179426
+     }
+
+**Key Components Explained:**
+
+- **single_sweeps**: This array contains data for each frequency sweep performed during the simulation.
+  Each entry in the array represents a set of results for a specific frequency.
+
+  - **f**: The frequency at which the sweep was conducted, measured in Hertz (Hz).
+
+  - **winding1** and **winding2**: These sections provide detailed results for each winding, including:
+
+    - **turn_losses**: The power losses in each turn of the winding.
+    - **flux**: The magnetic flux linked with the winding. The array
+      contains two values, representing the real and imaginary parts of the flux, respectively.
+    - **flux_over_current**: Represents the flux linkage per unit of current, also divided into
+      real and imaginary parts. This metric is crucial for understanding the efficiency of magnetic coupling in the transformer.
+    - **V**: Voltage across the winding, with the first value indicating the real part and the second the imaginary part.
+    - **I**: Current through the winding, with similar representation to voltage.
+
+  - **core_eddy_losses** and **core_hyst_losses**: These values represent the losses due to eddy currents and hysteresis in the core,
+    crucial for evaluating core material performance under alternating magnetic fields.
+- **total_losses**: This section summarizes the overall energy dissipation within the system, combining losses from various components.
+  It is broken down into several key areas:
+
+    - **winding1**: It indicates the total losses within every winding, which is the sum of power losses across all turns in this winding.
+
+    - **all_windings**: It aggregates the losses across all windings in the system.
+
+    - **core**: This repeats the total losses within the core.
```

Comment:
also explain the core parts. What when is the counter increased from `total_core_part_1` to `_2` and `_3` and so on?